.githooks/ktlintcommit

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+sleep 1
+git commit -n -a -m "Automatic commit for styling fixes"
```

Comment:
Is it still applicable or should be removed?

Reply:
Ill remove. Its left over files.